DatadogLogs/Sources/RemoteLogger.swift

```diff
@@ -110,6 +110,7 @@ internal final class RemoteLogger: LoggerProtocol {
         let tags = self.tags
         var logAttributes = attributes
         let isCrash = logAttributes?.removeValue(forKey: CrossPlatformAttributes.errorLogIsCrash) as? Bool ?? false
+        let errorFingerprint = logAttributes?.removeValue(forKey: Logs.Attributes.errorFingerprint) as? String
```

Comment:
question/ Shouldn't we check if `level` is `>= error` 🤔, so this is not applied to lower severities?

Reply:
No I don't think so, as you can pass errors to any level.  I could check that an error was passed in though?

Reply:
Ah, indeed 👍. All good 👌